packages/polling-controller/src/PollingController.test.ts

```diff
@@ -117,7 +126,7 @@ describe('PollingController', () => {
     });
   });
   describe('poll', () => {
```

Comment:
Nit: Typically, we use specific names for `describe`s — that is, we name them after the methods that we are testing. Instead, this file seems to name `describe`s more loosely. It seems like this would be confusing for anyone maintaining these tests, particularly `poll` here, as there is no method called "poll" (leaving the question of how these differ from `start` and `stop`). It would be nice to clean this up at some point.